ArmRobot/Assets/Scripts/Light/LightRotationRandomizer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using UnityEngine;
+
+public class LightRotationRandomizer : MonoBehaviour {
```

Comment:
If we switch this to a point light (rather than directional), then we don't need to worry about the light rotation. Then we could just randomize the number and location of the lights.